doc/changelog.rst

```diff
@@ -6,12 +6,90 @@ Changelog
 Motor 3.0
 ---------
 
-Motor 3.0 requires PyMongo 4.0+ and Python 3.7+.
-
+Motor 3.0 adds support for PyMongo 4.0+.  It inherits a number
+of breaking API changes from PyMongo 4.0.
```

Comment:
Let's link to the Motor migration guide here.

Reply:
Done